packages/orbit-components/src/Textarea/__tests__/index.test.js

```diff
@@ -1,108 +1,70 @@
-  const maxLength = 200;
-  const fullHeight = true;
-  const onChange = jest.fn();
-  const onFocus = jest.fn();
-  const onBlur = jest.fn();
-  const spaceAfter = SPACINGS_AFTER.NORMAL;
+describe("Textarea", () => {
+  it("should have expected DOM output", () => {
+    const name = "name";
+    const label = "Label";
+    const value = "value";
+    const placeholder = "placeholder";
+    const dataTest = "test";
+    const maxLength = 200;
+    const fullHeight = true;
+    const onChange = jest.fn();
+    const onFocus = jest.fn();
+    const onBlur = jest.fn();
+    const spaceAfter = SPACINGS_AFTER.NORMAL;
 
-  const component = shallow(
-    <Textarea
-      size={size}
-      name={name}
-      label={label}
-      value={value}
-      fullHeight={fullHeight}
-      placeholder={placeholder}
-      maxLength={maxLength}
-      help={<div>Something useful.</div>}
-      tabIndex={tabIndex}
-      readOnly
-      onChange={onChange}
-      onFocus={onFocus}
-      onBlur={onBlur}
-      dataTest={dataTest}
-      spaceAfter={spaceAfter}
-    />,
-  );
+    render(
+      <Textarea
+        name={name}
+        label={label}
+        value={value}
+        rows={4}
+        fullHeight={fullHeight}
+        placeholder={placeholder}
+        maxLength={maxLength}
+        help={<div>Something useful.</div>}
+        readOnly
+        onChange={onChange}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        dataTest={dataTest}
+        spaceAfter={spaceAfter}
+      />,
+    );
 
-  const area = component.find("Textarea__StyledTextArea");
-  const field = component.find("Textarea__Field");
+    const textarea = screen.getByRole("textbox");
 
-  it("should contain a label", () => {
-    expect(component.find("FormLabel").render().text()).toBe(label);
-  });
-  it("should contain a textarea", () => {
-    expect(area.exists()).toBe(true);
-  });
-  it("should have passed props", () => {
-    expect(area.prop("size")).toBe(size);
-    expect(field.prop("spaceAfter")).toBe(spaceAfter);
-    expect(area.prop("name")).toBe(name);
-    expect(area.prop("value")).toBe(value);
-    expect(area.prop("placeholder")).toBe(placeholder);
-    expect(area.prop("maxLength")).toBe(maxLength);
-    expect(area.prop("fullHeight")).toBe(fullHeight);
-    expect(area.render().prop("data-test")).toBe(dataTest);
-    expect(area.render().prop("readonly")).toBe(true);
-    expect(area.render().prop("tabindex")).toBe(tabIndex);
-    expect(component.find("Textarea__StyledTextArea").render().prop("attribs").name).toBe(name);
-  });
-  it("should contain FeedBack", () => {
-    expect(component.find(`FormFeedback`).exists()).toBe(true);
-  });
-  it("should execute onChange method", () => {
-    area.simulate("change");
-    expect(onChange).toHaveBeenCalled();
-  });
-  it("should execute onFocus method", () => {
-    area.simulate("focus");
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    expect(textarea).toHaveAttribute("tabindex", "0");
```

Comment:
You can leave setting `tabIndex` explicitly to `-1`, just to assert that setting it works as intended, like the Enzyme test did before, then split event handlers into a separate test without setting `tabindex` specifically (defaulting to `0`).